src/main/java/frc/robot/RobotContainer.java

```diff
@@ -188,16 +191,29 @@ private Trigger testTrig(Trigger t) {
   private void configureTestButtonBindsing() {
     testTrig(OI.getPOVButton(OI.Driver.DPAD_UP))
         .whileTrue(elevator.setElvPercent(OI.getAxisSupplier(OI.Driver.RightY).get()));
-    testTrig(OI.getPOVButton(OI.Driver.DPAD_RIGHT)).whileTrue(intake.intakeCommand());
-    testTrig(OI.getPOVButton(OI.Driver.DPAD_LEFT)).whileTrue(intake.outtakeCommand());
-    testTrig(OI.getButton(OI.Driver.RBumper)).whileTrue(intake.conveyorEject());
-    testTrig(OI.getButton(OI.Driver.LBumper)).whileTrue(intake.conveyorFeed());
-    // testTrig(usingKeyboard ? OI.getButton(OI.Keyboard.M) : OI.getTrigger(OI.Driver.RTrigger))
-    //     .whileTrue(climber.runRaw(Volts.of(3)));
-    // testTrig(usingKeyboard ? OI.getButton(OI.Keyboard.Comma) : OI.getTrigger(OI.Driver.LTrigger))
-    //     .whileTrue(climber.runRaw(Volts.of(-3)));
-    // testTrig(usingKeyboard ? OI.getButton(OI.Keyboard.Period) : OI.getButton(OI.Driver.Start))
-    //     .onTrue(climber.toggleJeopardy());
+    // testTrig(OI.getPOVButton(OI.Driver.DPAD_RIGHT)).whileTrue(intake.intakeCommand());
+    // testTrig(OI.getPOVButton(OI.Driver.DPAD_LEFT)).whileTrue(intake.outtakeCommand());
+    // testTrig(OI.getButton(OI.Driver.RBumper)).whileTrue(intake.conveyorEject());
+    // testTrig(OI.getButton(OI.Driver.LBumper)).whileTrue(intake.conveyorFeed());
+    // testTrig(OI.getButton(OI.Driver.X)).whileTrue(intake.extendPivotCommand());
+    // testTrig(OI.getButton(OI.Driver.Y)).whileTrue(intake.retractPivotCommand());
+
+    testTrig(OI.getButton(OI.Driver.LBumper)).onTrue(climber.engageServo());
+    testTrig(OI.getButton(OI.Driver.RBumper)).onTrue(climber.disengageServo());
+    testTrig(OI.getButton(OI.Driver.B))
+        .onTrue(climber.runClimber(ClimberConstants.kClimberAtCageSetpoint, 0));
```

Comment:
```suggestion
    testTrig(OI.getButton(OI.Driver.B)).onTrue(climber.extendToCage());
```
Looks like we already have a Command factory for this. Make runClimber() private again as well